src/tests/eolian/data/class_simple_ref.c

```diff
@@ -29,7 +29,7 @@ __eolian_class_simple_a_get_reflect(const Eo *obj)
    return eina_value_int_init(val);
 }
 
-EOAPI EFL_FUNC_BODY_CONST(efl_canvas_object_simple_a_get, int, 100);
+EAPI EAPI_WEAK EFL_FUNC_BODY_CONST(efl_canvas_object_simple_a_get, int, 100);
```

Comment:
@felipealmeida why using `EAPI EAPI_WEAK` here? Shouldn't it be an `EO_API` or something like that?

Reply:
Yeah I think we should use `EO_API EO_API_WEAK` here.

Reply:
Actually, as this is a test, we don't need any API setting, as it will only be used inside itself, it will not be exported/imported.

Reply:
Done